pyproject.toml

```diff
@@ -401,30 +404,6 @@ known-first-party = ["litestar", "tests", "examples"]
 ]
 "docs/examples/application_hooks/before_send_hook.py" = ["UP006"]
 "docs/examples/contrib/sqlalchemy/plugins/**/*.*" = ["UP006"]
-"docs/examples/tests/**/*.*" = [
```

Comment:
why we do this?

Reply:
the example tests have been moved into the tests namespace, this is unused